CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
 - Added: `selector-pseudo-element-case` rule.
 - Added: `value-keyword-case` rule.
 - Added: `selector-pseudo-class-case` rule.
+- Added: Handling of `.stylelintignore` file to add more ignored patterns
```

Comment:
1. Can you bump these to the top of the changelog please.
2. Lowercase the H and N of Handling and Notice respectively.
3. Add full stops.